node_openbsd.go

```diff
@@ -0,0 +1,36 @@
+package restic
+
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+func (node *Node) OpenForReading() (*os.File, error) {
+	file, err := os.OpenFile(node.path, os.O_RDONLY, 0)
+	if os.IsPermission(err) {
+		return os.OpenFile(node.path, os.O_RDONLY, 0)
```

Comment:
this is kinda useless... If you get permission denied, you just retry the exact same thing with the exact same flags? why would it work the second time? If OpenBSD doesn't have NOATIME, then this function should be simplified to
```golang
func (node *Node) OpenForReading() (*os.File, error) {
  return os.OpenFile(node.path, os.O_RDONLY, 0)
}
```